docs/examples/viz_pbr_interactive.py

```diff
@@ -0,0 +1,353 @@
+    global pbr_params, sphere
+    pbr_params['anisotropy'] = slider.value
+    sphere.GetProperty().SetAnisotropy(pbr_params['anisotropy'])
+
+
+def change_slice_anisotropy_direction_x(slider):
+    global doa, normals, sphere
+    doa[0] = slider.value
+    tangents = tangents_from_direction_of_anisotropy(normals, doa)
+    tangents_to_actor(sphere, tangents)
+
+
+def change_slice_anisotropy_direction_y(slider):
+    global doa, normals, sphere
+    doa[1] = slider.value
+    tangents = tangents_from_direction_of_anisotropy(normals, doa)
+    tangents_to_actor(sphere, tangents)
+
+
+def change_slice_anisotropy_direction_z(slider):
+    global doa, normals, sphere
+    doa[2] = slider.value
+    tangents = tangents_from_direction_of_anisotropy(normals, doa)
+    tangents_to_actor(sphere, tangents)
+
+
+def change_slice_anisotropy_rotation(slider):
+    global pbr_params, sphere
+    pbr_params['anisotropy_rotation'] = slider.value
+    sphere.GetProperty().SetAnisotropyRotation(
+        pbr_params['anisotropy_rotation'])
+
+
+def change_slice_coat_strength(slider):
+    global pbr_params, sphere
+    pbr_params['coat_strength'] = slider.value
+    sphere.GetProperty().SetCoatStrength(pbr_params['coat_strength'])
+
+
+def change_slice_coat_roughness(slider):
+    global pbr_params, sphere
+    pbr_params['coat_roughness'] = slider.value
+    sphere.GetProperty().SetCoatRoughness(pbr_params['coat_roughness'])
+
+
+def change_slice_base_ior(slider):
+    global pbr_params, sphere
+    pbr_params['base_ior'] = slider.value
+    sphere.GetProperty().SetBaseIOR(pbr_params['base_ior'])
+
+
+def change_slice_coat_ior(slider):
+    global pbr_params, sphere
+    pbr_params['coat_ior'] = slider.value
+    sphere.GetProperty().SetCoatIOR(pbr_params['coat_ior'])
+
+
+"""
+Last, but not least, we define the following function to help us to reposition
+the UI elements every time we resize the window.
+"""
+
+
+def win_callback(obj, event):
+    global control_panel, size
+    if size != obj.GetSize():
+        size_old = size
+        size = obj.GetSize()
+        size_change = [size[0] - size_old[0], 0]
+        control_panel.re_align(size_change)
+
+
+"""
+Let's fetch a skybox texture from the FURY data repository.
+"""
+
+fetch_viz_cubemaps()
+
+"""
+The following function returns the full path of the 6 images composing the
+skybox.
+"""
+
+textures = read_viz_cubemap('skybox')
+
+"""
+Now that we have the location of the textures, let's load them and create a
+Cube Map Texture object.
+"""
+
+cubemap = load_cubemap_texture(textures)
+
+"""
+The Scene object in FURY can handle cube map textures and extract light
+information from them, so it can be used to create more plausible materials
+interactions. The ``skybox_tex`` parameter is the one performing the previously
+described process. On the other hand, the ``render_skybox`` parameter toggles
+the rendering of the skybox.
+"""
+
+scene = window.Scene(skybox_tex=cubemap, render_skybox=True)
```

Comment:
I would use one parameter. Can you explain why two parameters are necessary?

Reply:
I will add an issue to remove the skybox since that might require extra functionality to notify the actors they should not use IBL anymore.

The update skybox should be easier since the light information is only calculated when we render the scene. However, I think this should come in a future PR too.